components/edit-collective/sections/HostVirtualCards.js

```diff
@@ -118,6 +136,9 @@ const HostVirtualCards = props => {
       offset,
       state,
       merchantAccount: { slug: merchant },
+      collectiveAccounts: collectiveAccounts
+        ? collectiveAccounts.split(',').map(collectiveAccount => ({ slug: collectiveAccount }))
```

Comment:
:cherries: Include `id` to take advantage of data loader caching if implemented.

Reply:
Done, let me know if you see any further improvements. 😄